contracts/utils/ERC721AQueryablePolicies.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "erc721a/contracts/extensions/ERC721AQueryable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/StringsUpgradeable.sol";
+
+/**
+ * @dev Library containing {ERC721AQueryable}-related helper methods for writing {ITablelandController.Policy}s.
+ */
+library ERC721AQueryablePolicies {
```

Comment:
these helper libs are split up so that people only need to compile the code that they need